docs/developers/Arch.md

```diff
@@ -55,19 +55,19 @@
 
 | Field                      | Type         | Null | Key | Default | Extra          |
 |----------------------------|--------------|------|-----|---------|----------------|
-| CALL_CACHING_HASH_ENTRY_ID | int(11)      | NO   | PRI | NULL    | auto_increment |
+| CALL_CACHING_HASH_ENTRY_ID | bigint(20)   | NO   | PRI | NULL    | auto_increment |
```

Comment:
TOL: This documentation seems strangely specific when it comes to:
* mysql-specific table structure

And strangely non-specific when it comes to:
* the actual table names
* what the tables are doing
* which fields are FKs
  * ... and which fields in which other tables those FKs link to

Reply:
Yeah this kind of surprised me too but I only went in here to adjust the PK datatype and fix a bogus reference to the PK.